src/main/java/com/commercetools/sync/customers/utils/CustomerReferenceResolutionUtils.java

```diff
@@ -61,24 +60,28 @@
    *   </tbody>
    * </table>
    *
-   * <p><b>Note:</b> The {@link CustomerGroup} and {@link Type} references should be expanded with a
-   * key. Any reference that is not expanded will have its id in place and not replaced by the key
-   * will be considered as existing resources on the target commercetools project and the library
-   * will issues an update/create API request without reference resolution.
+   * <p><b>Note:</b> The {@link CustomerGroup} and {@link Type} references should contain Id in the
+   * map(cache) with a key value. Any reference that is not available in the map will have its id in
+   * place and not replaced by the key. This reference will be considered as existing resources on
```

Comment:
```suggestion
   * map(cache) with a key value. Any reference, which have its id in place and not replaced by the key, it would not 
   * be found in the map. In this case, this reference will be considered as existing resources on 
```